aria/vllm/aria.py

```diff
@@ -718,7 +583,7 @@ def __init__(
     ):
         super().__init__()
         config = vllm_config.model_config.hf_config
-        cache_config = vllm_config.cache_config
+        vllm_config.cache_config
```

Comment:
What's this line for?

Reply:
I will remove it. This change was made by the formatter because the `cache_config` is not currently used.